core/src/main/scala/filodb.core/query/QueryUtils.scala

```diff
@@ -0,0 +1,25 @@
+package filodb.core.query
+
+/**
+ * Storage for utility functions.
+ */
+object QueryUtils {
+  private val regexChars = Array('.', '?', '+', '*', '|', '{', '}', '[', ']', '(', ')', '"', '\\')
+  private val regexCharsMinusPipe = (regexChars.toSet - '|').toArray
+
+  /**
+   * Returns true iff the argument string contains any special regex chars.
+   */
+  def containsRegexChars(str: String): Boolean = {
+    str.exists(regexChars.contains(_))
```

Comment:
Why ``exists`` when the original implementation was ``forall``?

Reply:
Seemed more scala-thonic 🤷 
I can change it back if there's some weird difference between the two and/or the consensus is that `forall` is more readable?

Reply:
they aren't same , ``exists`` returns true of at least one is true and ``forall`` requires all to be ``true``

```
scala> val l = List(1, 2, 3, 4)
l: List[Int] = List(1, 2, 3, 4)

scala> l.exists(_ > 2)
res5: Boolean = true

scala> l.forall(_ > 2)
res6: Boolean = false

```

Reply:
I see the difference after speaking offline, totally missed the `!`, I agree the refactor is more readable than original .